### DISCUSSION TITLE
How to move PTZ Camera by C#

### DISCUSSION THREAD

**Discussion by andsouza96** [AxisCommunications/discussions#24] [Axis Windows SDKs] [answered]
I would like to know how I can move PTZ Camera through code C#?
I can enable the Joystick mode but don't find a way to move with parameters or methods.
Is there this feature to Axis Media Control SDK?

Thank you.

**Comment by pataxis** [comment]
Hi @andsouza96 , I have looked for someone that could help answer this question but it seems many are still on vacation so just wanted to give a heads up that it may take some more weeks.

**Comment by anders-axis** [comment, marked as answer]
You can control PTZ using the VAPIX API. It is not directly supported in the AXIS Media Control SDK, but it should be fairly simple to write the code to make the HTTP requests from C#, e.g. using WebClient or similar.
https://www.axis.com/vapix-library/subjects/t10175981/section/t10036011/display?section=t10036011-t10004639
